backend/src/controllers/role/entitlementTypeController.ts

```diff
@@ -0,0 +1,42 @@
+import { prisma, errorResponse } from "../../utils";
+import { Request, Response } from "express";
+
+const initEntTypes = [
```

Comment:
```suggestion
const initEntitlementTypes = [
```
You should name it in a way it's easy to understand.